scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -6,57 +6,37 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+Workload partitioning segregates compute node CPU resources into distinct CPU sets. The primary objective is to keep platform pods on the specified cores to avoid interrupting the CPUs the customer workloads are running on. Workload partitioning isolates {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. This ensures that the remaining CPUs in their deployment remain untouched and available exclusively for their own workloads. The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
 
-The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
-With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
-These pods operate normally within the minimum size CPU configuration.
-Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+In the context of enabling workload partitioning and managing CPU resources effectively, nodes within the cluster must identify themselves as supporting a new resource type called `management.workload.openshift.io/cores` and accurately report their CPU capacity. Workload partitioning can be enabled during cluster installation only by adding the additional field: `cpuPartitioningMode` to the `install-config.yaml` file.
```

Comment:
We should change this a bit so customers know what is being done automatically and what they might need to be aware of regarding the nodes.

Something like this maybe?

>In the context of enabling workload partitioning and managing CPU resources effectively, nodes that are not configured correctly will not be allowed to join the cluster via a node admission webhook. When the workload partitioning feature is enabled, the Machine Config Pools for `master` and `worker` will be supplied with configurations for nodes to use. Adding new nodes to these pools will make sure they are correctly configured before joining the cluster. 
>
>Currently, it is required that nodes are homogenous per Machine Config Pool in order to ensure correct CPU Affinity is set across all nodes with in that pool. Once admitted nodes within the cluster will identify themselves as supporting a new resource type called `management.workload.openshift.io/cores` and accurately report their CPU capacity. Workload partitioning can be enabled during cluster installation only by adding the additional field: `cpuPartitioningMode` to the `install-config.yaml` file.